bindings/py/tests/nupic_random_test.py

```diff
@@ -44,21 +36,17 @@
 class TestNupicRandom(unittest.TestCase):
 
 
-  @unittest.skipUnless(
-    capnp, "pycapnp is not installed, skipping serialization test.")
```

Comment:
keep the skipUnless logic? Could be nice to have a variable serialization (former capnp) and if set to None, the tests are skipped.